uc-crux-llvm/uc-crux-llvm.cabal

```diff
@@ -0,0 +1,157 @@
+Name:          uc-crux-llvm
+Version:       0.1
+Author:        Galois Inc.
+Maintainer:    Langston Barrett, Scott Moore, Tristan Ravitch
+Copyright:     (c) Galois, Inc 2021
+License:       BSD3
+License-file:  LICENSE
+Build-type:    Simple
+Cabal-version: >= 1.10
+Category:      Language
+Synopsis:      A bugfinding tool for C programs, using under-constrained symbolic execution.
+
+library
+
+  hs-source-dirs: src
+
+  exposed-modules:
+    UCCrux.LLVM.Classify
+    UCCrux.LLVM.Classify.Poison
+    UCCrux.LLVM.Classify.Types
+    UCCrux.LLVM.Config
+    UCCrux.LLVM.Constraints
+    UCCrux.LLVM.Context.App
+    UCCrux.LLVM.Context.Function
+    UCCrux.LLVM.Context.Module
+    UCCrux.LLVM.Cursor
+    UCCrux.LLVM.Errors.Panic
+    UCCrux.LLVM.Errors.Unimplemented
+    UCCrux.LLVM.FullType
+    UCCrux.LLVM.FullType.CrucibleType
+    UCCrux.LLVM.FullType.MemType
+    UCCrux.LLVM.FullType.ModuleTypes
+    UCCrux.LLVM.FullType.Type
+    UCCrux.LLVM.Logging
+    UCCrux.LLVM.Main
+    UCCrux.LLVM.PP
+    UCCrux.LLVM.Run.Explore
+    UCCrux.LLVM.Run.Loop
+    UCCrux.LLVM.Run.Result
+    UCCrux.LLVM.Run.Simulate
+    UCCrux.LLVM.Setup
+    UCCrux.LLVM.Setup.Monad
+    UCCrux.LLVM.Shape
+    UCCrux.LLVM.Stats
+
+  build-depends:
+    ansi-terminal,
+    bytestring,
+    base >= 4.8 && < 4.15,
+    bv-sized,
+    config-schema,
+    containers,
+    crucible,
+    crucible-llvm,
+    crux,
+    crux-llvm,
+    data-binary-ieee754,
+    directory,
+    filepath,
+    lens,
+    llvm-pretty,
+    llvm-pretty-bc-parser,
+    lumberjack,
+    mtl,
+    panic,
+    parameterized-utils,
+    prettyprinter >= 1.7.0,
+    process,
```

Comment:
Interesting - I didn't see this used, but might just be tired

Reply:
Oh nope, that's just a result of copy pasting the crux-llvm cabal file... Will remove.

Reply:
FYI: https://github.com/GaloisInc/crucible/pull/678/files

Reply:
:disappointed: will incorporate...

Reply:
Yeah, there's a lot of unused packages. I want to add `-Werror=unused-packages` to each stanza before merging.

Reply:
https://github.com/GaloisInc/crucible/pull/673/commits/6612eea410a77373969fd14a5181681001b49811

Reply:
https://github.com/GaloisInc/crucible/pull/673/commits/c998491c75cd026e0528ae56bd93b29bc6ec14c2